lib/config.js

```diff
@@ -154,6 +154,8 @@ function getLocalConfig(thisConfig, directory) {
 
         if (personalConfig) {
             return ConfigOps.merge(config, personalConfig);
+        } else if (thisConfig.options.rules && thisConfig.options.rules !== {}) {
```

Comment:
Yeah, that's a better way to do it. Also, can you add in a comment explaining when this condition will be hit?